src/main/java/com/jcraft/jsch/UserAuthKeyboardInteractive.java

```diff
@@ -129,7 +129,8 @@ public boolean start(Session session) throws Exception {
           byte[][] response = null;
 
           if (password != null && prompt.length == 1 && !echo[0]
-              && prompt[0].toLowerCase().indexOf("password:") >= 0) {
+              && (prompt[0].toLowerCase().indexOf("password:") >= 0
+            		 || prompt[0].toLowerCase().indexOf("password") >= 0)){
```

Comment:
In #302 you mention that this change is needed in order to support a server that has a prompt of `Enter password:`.
However, the string `Enter password:` already should match `prompt[0].toLowerCase().indexOf("password:") >= 0`, so I'm not sure I understand the purpose of this change (see below)?

```
jshell> String foo = "Enter password:";
foo ==> "Enter password:"

jshell> if (foo.toLowerCase().indexOf("password:") >= 0) System.out.println("matches"); else throw new Exception("fail");
matches

jshell> 
```

Reply:
I am not able to reproduce this issue in my test system so cannot test it. We can leave the changes done in UserAuthKeyboardInteractive.java for now. Please let me know if I need to revert the changes and commit again. Thanks.

Reply:
The or also overlaps, you could just remove the colon..